sections/tooling/babel-macro.md

```diff
@@ -10,6 +10,14 @@ import styled from 'styled-components/macro'
 const Thing = styled.div`
   color: red;
 `
+
+import { createGlobalStyle } from 'styled-components/macro'
```

Comment:
Can you add this to the first import? The code is technically invalid otherwise

Reply:
You're right. I was concerned by the readability of the whole thing, but the example is still concise and simple enough anyway. Fixed.